mlflow/store/db/utils.py

```diff
@@ -81,9 +82,12 @@ def make_managed_session():
         except MlflowException:
             session.rollback()
             raise
+        except sqlite3.IntegrityError as e:
+            session.rollback()
+            raise MlflowException(message=str(e), error_code=RESOURCE_ALREADY_EXISTS)
```

Comment:
`RESOURCE_ALREADY_EXISTS` http error respond code is 400, it should prevent client retries, but in my testing it still retries. What's the reason ?

Reply:
Can you share the code you used?

Reply:
is 400 responding code preventing retries ?

Reply:
Can we stringify `message` in `MlflowException`?

Reply:
> Can we stringify message in the MlflowException constructor?
this sounds a little strange. The argument name is "message" which means it should be a text.

Reply:
It does, but I think it's better than replace all `MlflowException(error_instance)` to `MlflowException(str(e))`.

Reply:
If we used a type checker like mypy, we could prevent passing a non-string value to `MlflowException` but we currently don't use mypy.